planos/005-exploration-gabwindow/feedback-gabwindow/builds/fase1/apply-gab-feedback.js

```
@@ -0,0 +1,31 @@
+  "frontend/data/Map045.json": {
+    interactions: [[5,4,21],[7,1,9],[8,2,11],[8,2,26],[8,2,31],[8,2,36],[8,2,41],[8,2,46],[8,2,51],[8,2,56],[8,3,11],[8,4,12],[8,4,27],[8,4,42],[8,4,47],[8,4,52],[8,4,92],[8,4,122],[8,6,11],[12,1,12],[12,2,12],[12,3,12],[20,2,11]],
+    force: [[5,4,21],[7,1,9],[8,2,11],[8,3,11],[8,4,12],[8,4,122],[8,6,11],[12,1,12],[12,2,12],[12,3,12],[20,2,11]]
+  }
+};
+
+for (const [file, spec] of Object.entries(targets)) {
+  const data = JSON.parse(fs.readFileSync(file, "utf8"));
+  const force = new Set(spec.force.map(x => x.join(":")));
+  for (const [eventId, pageNo, index] of spec.interactions) {
+    const command = data.events[eventId].pages[pageNo - 1].list[index];
+    if (command.code !== 357 || command.parameters[1] !== "GabTextOnly") throw new Error(`Unexpected Gab at ${file} E${eventId}/P${pageNo}/${index}`);
```

ESLint

[error] Replace `"GabTextOnly"` with `'GabTextOnly'`

(prettier/prettier)